### PR TITLE
FYLE-85zrk1mt8 Code coverage : Singular pipe

### PR DESCRIPTION
**Pull request by CODEX108**

### CODE REVIEW

File: src/app/shared/pipes/singular.pipe.spec.ts [hunk 1]
```diff
@@ -1,8 +1,28 @@
 import { SingularPipe } from './singular.pipe';
 
 describe('SingularPipe', () => {
+  let pipe: SingularPipe;
```
Comment (jayfyle) [inline src/app/shared/pipes/singular.pipe.spec.ts:4]: wherever using the name pipe, replace with `singularPipe`

File: src/app/shared/pipes/singular.pipe.spec.ts [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+
+  const string1 = 'Locations';
+  const string2 = 'Testing';
+
   it('create an instance', () => {
-    const pipe = new SingularPipe();
     expect(pipe).toBeTruthy();
   });
+
+  it(`should remove "s" from the end of a "${string1}"`, () => {
```
Comment (jayfyle) [inline src/app/shared/pipes/singular.pipe.spec.ts:17]: Include function name in the spec description. Change the spec description to `should remove s from the end of a string`, no need to use template string.

**Review by jayfyle — APPROVED** [review]
Minor comments

File: src/app/shared/pipes/singular.pipe.spec.ts [hunk 3]
```diff
@@ -1,8 +1,28 @@
 import { SingularPipe } from './singular.pipe';
 
 describe('SingularPipe', () => {
+  let pipe: SingularPipe;
+
+  beforeEach(() => {
```
Comment (Dimple16) [inline src/app/shared/pipes/singular.pipe.spec.ts:6]: why use `beforeEach`? 
let's directly use - `const pipe = new SingularPipe()`

File: src/app/shared/pipes/singular.pipe.spec.ts (hunk 1 above)
Comment (Dimple16) [inline src/app/shared/pipes/singular.pipe.spec.ts:4]: this won't be required

**Review by Dimple16 — CHANGES_REQUESTED** [review]

**Review by jayfyle — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**